src/test/ui/const-generics/slice-const-param-mismatch.stderr

```diff
@@ -0,0 +1,38 @@
+warning: the feature `const_generics` is incomplete and may cause the compiler to crash
+  --> $DIR/slice-const-param-mismatch.rs:1:12
+   |
+LL | #![feature(const_generics)]
+   |            ^^^^^^^^^^^^^^
+   |
+   = note: `#[warn(incomplete_features)]` on by default
+
+error[E0308]: mismatched types
+  --> $DIR/slice-const-param-mismatch.rs:9:35
+   |
+LL |     let _: ConstString<"Hello"> = ConstString::<"World">;
+   |                                   ^^^^^^^^^^^^^^^^^^^^^^ expected `"Hello"`, found `"World"`
+   |
+   = note: expected type `ConstString<>`
```

Comment:
btw @varkor, this is a rather unfortunate diagnostic... do we have an issue for that?

Reply:
There is a case for it: #61395.

Reply:
Ah excellent, thanks!